app/controllers/public_recipes_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PublicRecipesController < ApplicationController
+  def index
+    @public_recipes = Recipe.includes(:user, :recipe_foods).where(public: true).order(created_at: :desc)
+  end
```

Comment:
- [ ] Just so you know, public recipes should be available to the public (anyone), I logged out, and then I created another account. I did not see the general (public) recipe I created using another account. Kindly fix this issue as well.

Reply:
Hello, @FuadNabi and @jorgegoco thank you for having the time to explain the app to me on zoom. Your project is complete and it will be approved by the next reviewer 👍.